style.css

```diff
@@ -93,3 +93,334 @@ header {
+  line-height: 21px;
+  color: #1a2236;
+}
+
+.about > button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  width: 288px;
+  height: 50px;
+  background: #1a2236;
+  padding: 13px 60px;
+  color: #fff;
+  border: none;
+}
+
+.languages {
+  display: flex;
+  flex-direction: column;
+  width: 289px;
+  height: 289px;
+  background: #1c528a;
+  color: #fff;
+  gap: 21px;
+  padding-left: 15px;
+  padding-top: 30px;
+  background-image: url("./images/hash.svg");
+  background-position: bottom right;
+  background-repeat: no-repeat;
+}
+
+.languages > h3 {
+  width: 116px;
+  height: 30px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 20px;
+  line-height: 30px;
+  margin-left: 30px;
+}
+
+.languages > ul {
+  width: 178px;
+  height: 135px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 27px;
+  letter-spacing: -0.0525em;
+  list-style-type: disc;
+  margin-left: 50px;
+}
+
+.rectangle {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.framework {
+  display: flex;
+  flex-direction: column;
+  width: 289px;
+  height: 289px;
+  background: #e2505c;
+  color: #fff;
+  padding-left: 15px;
+  gap: 21px;
+  padding-top: 30px;
+  background-image: url("./images/Rectangle.svg");
+  background-repeat: no-repeat;
+  background-position: bottom right;
+}
+
+.framework > h3 {
+  width: 132px;
+  height: 30px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 20px;
+  line-height: 30px;
+  margin-left: 30px;
+}
+
+.framework > ul {
+  width: 118px;
+  height: 135px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 27px;
+  letter-spacing: -0.0525em;
+  list-style-type: disc;
+  margin-left: 50px;
+}
+
+.skills {
+  display: flex;
+  flex-direction: column;
+  width: 289px;
+  height: 289px;
+  background: #febf52;
+  color: #fff;
+  gap: 21px;
+  padding-top: 30px;
+  background-image: url("./images/bulb.svg");
+  background-repeat: no-repeat;
+  background-position: bottom right;
+}
+
+.skills > h3 {
+  width: 55px;
+  height: 30px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 20px;
+  line-height: 30px;
+  margin-left: 33px;
+}
+
+.skills > ul {
+  width: 88px;
+  height: 135px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 27px;
+  letter-spacing: -0.0525em;
+  list-style-type: disc;
+  margin-left: 50px;
+}
+
+.about > button:hover {
+  cursor: pointer;
+  background-color: #fff;
+  color: #1a2236;
+  border: 1px solid #1a2236;
+}
+
+.btn:hover {
+  cursor: pointer;
+  background-color: #fff;
+  color: #e2505c;
+  border: 1px solid #e2505c;
+}
+
+@media only screen and (max-width: 600px) {
+  body {
+    background-color: lightblue;
+  }
```

Comment:
- [ ] Kindly remove this media query since it's modifying the color of your background as it's not seen on the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) :recycle: 

- Your design:
![yourImage](https://user-images.githubusercontent.com/97834160/205317494-3de0e096-a749-423e-bde3-84fcffc1ee5f.png)


- Figma design:

![neededBackground](https://user-images.githubusercontent.com/97834160/205316218-f03275ba-0d5d-43a2-81f7-f85612021bda.png)